File: Code_Python/sentimental-mario-more/mario.py
# Function to get a valid integer between 1 and 8 inclusive
def get_int():
    while True:
        try:
            height = int(input("Height: "))
            if 1 <= height <= 8:
                return height
        except ValueError:
            pass
        print("Please enter a valid integer between 1 and 8.")


# Get the height of the pyramid
height = get_int()

# Generate the half-pyramids
for i in range(1, height + 1):
    # Left pyramid
    print(" " * (height - i), end="")  # Print spaces for alignment
    print("#" * i, end="")  # Print left hashes

    # Gap of two spaces
    print("  ", end="")

    # Right pyramid
    print("#" * i)

File: Code_C/mario-less/mario.c
int main(void)
{
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0);

    for (int i = 0; i < height; i++)
    {
        // Print spaces for right alignment
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }

        // Print hashes for the pyramid
        for (int k = 0; k <= i; k++)
        {
            printf("#");
        }

        // **Removed the trailing space**

        // Print a new line
        printf("\n");
    }
}
